package/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ add_executable(prmon src/prmon.cpp
 					 src/countmon.cpp 
 					 src/wallmon.cpp
 					 src/memmon.cpp)
-target_include_directories(prmon PRIVATE ${RapidJSON_INCLUDE_DIR})
+target_link_libraries(prmon PRIVATE nlohmann_json::nlohmann_json)
```

Comment:
Oddly, this line causes a failure when building on Ubuntu18 with the native installation of Nlohmann's JSON (`nlohmann-json-dev`).

Removing it seems to work find and not prejudice the CentOS7 build, when the library is taken from EPEL (`json-devel`).

This might be because the Ubuntu18 version is surprisingly old, 2.1.1, vs. the C7 3.0.0.